Task7/task7.html

```diff
@@ -0,0 +1,164 @@
+                        <div class="myBar3"></div>
+                    </div>
+                    60%
+                </div>
+            </div>
+
+            <div class="hl"></div>
+
+            <div class="row1">
+                <div class="column">
+                    <h1>2000 - 2008</h1>
+                    Jr. Graphic Designer |<span class="link2">ABC Inc.</span>  
+                </div>
+                <div class="column columnCenter">
+                    <h1>2008 - 2010</h1>
+                    Graphic Specialist | <span class="link2">GRAPHICSTUDID</span>
+                </div>
+                <div class="column">
+                    <h1>2010 - Present</h1>
+                    Creative Director | <span class="link2">DESIGN ARTS</span>
+                </div>
+                  
+            </div>
+        </div>
+
+    </div>
+    
+    <div class="div1">
+        <img id="imgLogo" class="img7" src ="imges/7.png"/>
+        <img id="theImage" class="img9" src ="imges/9.png"/> 
+        <div class="container" onclick="myFunction(this)">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+        </div>  
+    </div> 
+    <div class="div3">
+    <div class="divHelloTxt">
+            <span>Hello,</span><br>
+            Donec commodo leo est quis eleifend urna pellentesque eget. Proin scelerisque odio en nisi viverra, nec ullamcorper augue sagittis. Proin faucibus lorem nisl,
+    </div>   
+    </div>
+    
+    
+    <div class="div2">
+        <img src="imges/3.png" class="img3">
+        <img src="imges/4.png" class="img4">
+        <img src="imges/5.png" class="img5">
+        <img src="imges/6.png" class="img6">
+        <div class="readMore">
+            <a href="#">ــــــــ   read more</a>
+        </div>
+        
+        <div class="row"> 
+            <img class="imge" src ="imges/a.png"/>
+            <img class="imge" src ="imges/b.png"/><br>
+            <img class="imge" src ="imges/c.jpg"/>
+            <img class="imge" src ="imges/d.png"/>
+        </div>
+
+        <div class="div4">
+            
+            <span style="letter-spacing: 5px;font-weight: bold;">LOADING </span><br>
+            
+            <img class="img8" src ="imges/8.png"/>
+            <h1>
+                1.800.123.456
+            </h1>
+
+            <h6 style="color: #adadad;">
+                Donec commodo leo est quis eleifend urna pellentesque eget. 
+                Proin scelerisque odio eu nisi viverra, nec ullamcorper augue sagittis. 
+                Proin faucibus lor  or <a class="link2" href="#">contactme@mysite.com</a><br><br>
+            </h6>  
+            <div class="row1">
+                <a class="bottomLink" href="#">FACEBOOK</a>
+                <a class="bottomLink centerLink" href="#">TWITTER</a>
+                <a class="bottomLink" href="#">DRIBBBLE</a>       
+            </div>
+
+        </div> 
+
+        <div class="vl"></div>
```

Comment:
<div class="vl"></div>
?
empty tag and class name v1 not clear

Reply:
this is for vertical line in bottom

Reply:
name class ti be verticalLine to be clear for every one

Reply:
I do it 
https://github.com/RahafSalah/FrontEndTraining/pull/6